examples/customresources/models/mybase/mybase.go

```diff
@@ -130,6 +135,13 @@ func (base *MyBase) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
-func (base *MyBase) Close(ctx context.Context) error {
-	return base.Stop(ctx, nil)
+func (myBase *MyBase) Properties(ctx context.Context, extra map[string]interface{}) (base.Properties, error) {
+	return base.Properties{
+		TurningRadiusMeters: myBaseTruningRadiusM,
```

Comment:
I appreciate your consistency, but I think `Truning` should be spelled differently. :smile: